### ISSUE TITLE
Update to use asset library in works?

### ISSUE THREAD

**Issue by Bazmundi** [iCEE-HAM/io_import_lipSync-blender2.8#4]
I think this is still a viable option for animation of bones.  With pose libary being dropped from blender, is there an update of this addon to use assets instead please?

**Comment by Sobollion** [comment]
Yeah need to rewrite it for 3.6+ blender, they throw away a legacy pose_library

Python: Traceback (most recent call last):
io_import_lipSync_Importer_2_93.py", line 450, in execute
    if scn.fpath!='': lipsyncerFlexRig()
io_import_lipSync_Importer_2_93.py", line 135, in lipsyncerFlexRig
    state_poselib = obj.pose_library
AttributeError: 'Object' object has no attribute 'pose_library'

**Comment by Sobollion** [comment]
We have two ways: 
build a workaroun thru an action (they still maintain a pose_markers) (that allows us using an old pose library datablock)
or do it from new asset poses
